src/styles.css

```diff
@@ -1,4 +1,5 @@
 /* You can add global styles to this file, and also import other style files */
 @import '~bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Use only one styling bootstrap or material

Reply:
I'm on my way of switching from bootstrap to Material Design.
Will remove it later.

Reply:
ok